UXP-Migration-Guide/README.md

```diff
@@ -180,7 +179,7 @@ While plugins can still use v1, many new features are only available on Photosho
 
 CSXS/CEP events are used to send events among extensions in an application, and among extensions in different applications.  
 
-**Photoshop DOM events**: You can send and receive events within an extension by attaching event listeners in the DOM of the plugin panel or modal, and then listening for changes in the Photoshop app by using the Photoshop API. 
+**Photoshop DOM events**: You can send and receive events within an extension by attaching event listeners in the DOM of the plugin panel or modal, and then listening for changes in the Photoshop app by using the Photoshop UXP API. 
```

Comment:
"... by using the Photoshop DOM API" I should've specified Photoshop *DOM* API instead of saying just Photoshop API here, but that's what we want rather than UXP API